components/dashboard/src/settings/Account.tsx

```diff
@@ -67,7 +67,7 @@ export default function Account() {
             </div>
             <h3 className="mt-12">Delete Account</h3>
             <p className="text-base text-gray-500 pb-4">This action will remove all the data associated with your account in Gitpod.</p>
-            <button className="border-red-600 text-red-600 bg-white hover:border-red-800 hover:text-red-800" onClick={() => setModal(true)}>Delete Account</button>
+            <button className="warn" onClick={() => setModal(true)}>Delete Account</button>
```

Comment:
**thought**: More button variants will probaly emerge in the long run but I think this would fit better as a _danger_ button, not warn. It should be fine as _warn_ as _for the time being_ we need this once in _Account_. Danger usually describes a destructive action that can't be undone like deleting something.

Reply:
good point, `danger` sounds better. ☢️